src/common/Input/Input.js

```diff
@@ -38,90 +48,89 @@ const Input = React.forwardRef(
       type,
       value,
       withoutBorder,
-      wrapperClassName
+      wrapperClassName,
+      validationRules: rules
     },
     ref
   ) => {
+    ref = useRef()
```

Comment:
I don't understand why you reassign the `ref` here. I believe in this case we won't be able to use the ref outside this component